include/config/config_game.h

```diff
@@ -25,4 +25,8 @@
  */
 // #define FW_SPLIT_AGE
 
+ /* Keep pause delay. Inacurate emulators have can potentially crash if enabled. */
```

Comment:
the crash isn't from the filter but from the OoB read on the minimap right? This comment should be like ``/* Apply the anti-aliasing filter for the background of the pause menu. This adds a delay when pausing. */``